static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -138,7 +154,19 @@ export function InviteBanner({missingMembers, onSendInvite, organization}: Props
           <Button
             priority="primary"
             size="xs"
-            // TODO(cathy): open up invite modal
+            onClick={() =>
+              openModal(
+                deps => (
+                  <InviteMissingMembersModal
+                    {...deps}
+                    organization={organization}
+                    missingMembers={missingMembers.users}
```

Comment:
Renaming variables in this manner makes the code very hard to grok across different files, even with type hints. You should pass in the entire `missingMember` object.

Reply:
Doubly confusing with `missingUsers` in the `SeeMoreCard` above.